artifactory/utils/transfersettings.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+const transferSettingsFile = "transfer.conf"
+
+type TransferSettings struct {
```

Comment:
I think that we should use a shared mutex for reading and writing to the new configuration file.